src/js/blocks/panel/block.js

```diff
@@ -238,26 +230,27 @@ registerBlockType( 'uri-cl/panel', {
 			return (
 				<InspectorControls>
 					<PanelBody>
-					
+
 						<PanelRow>
 							<BaseControl
 								label={ __( 'Format' ) }
-								help={ __( 'To increase performance, super panel previews will appear simplified in the editor window.' )}
+								help={ __( 'To increase performance, super panel previews will appear simplified in the editor window.' ) }
+								id="panel-format"
 							>
 								<ButtonGroup aria-label={ __( 'Panel Format' ) }>
 									{ [ 'default', 'super' ].map( ( value ) => {
 										const capitalizedValue = value.charAt( 0 ).toUpperCase() + value.slice( 1 );
 										const key = ( 'default' === value ) ? '' : value;
-										const format = ( undefined == attributes.format ) ? '' : attributes.format;
-										const isSelected = ( key === format );
+										const format = ( undefined === attributes.format ) ? '' : attributes.format;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected use of undefined.](https://app.codacy.com/manual/uriweb/uri-component-library/pullRequest?prid=4811013)